napari/components/_tests/test_viewer_model.py

```diff
@@ -785,7 +785,6 @@ def test_status_tooltip(Layer, data, ndim):
     layer = Layer(data)
     viewer.layers.append(layer)
     viewer.cursor.position = (1,) * ndim
-    viewer._on_cursor_position_change()
```

Comment:
I removed this method as it's only setting `Layer.position`, which seems to have been deprecated, then removed, but we're still actually setting it in this callback.

Should this test assert anything? Or is it just checking for crashes. Given that we're using `qthrottled`, should we wait for some condition? Or does the first change always succeed?